src/pointer.rs

```diff
@@ -1130,6 +1597,45 @@ mod tests {
         }
     }
 
+    #[test]
+    #[allow(clippy::useless_asref)]
```

Comment:
Worth a short comment as to why this is a false positive

Reply:
👍